presidio-analyzer/tests/test_recognizer_registry_provider.py

```diff
@@ -35,8 +36,24 @@ def test_recognizer_registry_provider_configuration_file():
     assert [recognizer.supported_language for recognizer in recognizer_registry.recognizers if recognizer.name == "ItFiscalCodeRecognizer"] == ["en", "es"]
     assert [recognizer.supported_language for recognizer in recognizer_registry.recognizers if recognizer.name == "CreditCardRecognizer"] == ["en"]
     assert [recognizer.supported_language for recognizer in recognizer_registry.recognizers if recognizer.name == "ExampleCustomRecognizer"] == ["en", "es"]
-    snpanish_recognizer = [recognizer for recognizer in recognizer_registry.recognizers if recognizer.name == "ExampleCustomRecognizer" and recognizer.supported_language == "es"][0]
-    assert snpanish_recognizer.context == ["tarjeta", "credito"]
+    spanish_recognizer = [recognizer for recognizer in recognizer_registry.recognizers if recognizer.name == "ExampleCustomRecognizer" and recognizer.supported_language == "es"][0]
+    assert spanish_recognizer.context == ["tarjeta", "credito"]
+
+def test_recognizer_registry_provider_configuration_file_load_predefined(mandatory_recognizers):
+    this_path = Path(__file__).parent.absolute()
+    test_yaml = Path(this_path, "conf/test_recognizers.yaml")
+    provider = RecognizerRegistryProvider(test_yaml)
+    recognizer_registry = provider.create_recognizer_registry()
+    assert recognizer_registry.supported_languages == ["en"]
+    assert recognizer_registry.global_regex_flags == 26
+    assert len(recognizer_registry.recognizers) > 2
```

Comment:
Let's discuss if the nlp recognizer should be here in this phase or not. I assume >2 is because of the nlp recognizer